.gitignore

```diff
@@ -1,6 +1,7 @@
 *.class
```

Comment:
Can you also add the .idea folder here. Gradle metadata doesn't belong in the repo.

Reply:
Sure